keras_cv/utils/preprocessing.py

```diff
@@ -64,3 +65,96 @@ def _unwrap_value_range(value_range, dtype=tf.float32):
     min_value = tf.cast(min_value, dtype=dtype)
     max_value = tf.cast(max_value, dtype=dtype)
     return min_value, max_value
+
+
+def transform(
```

Comment:
Assume this method is copied from core keras, should we consider expose this as __internal__ method?

Reply:
I discussed this with Matt and we decided to just copy paste it as we want the freedom to make changes to it going forward.  We could expose it in __internal__ - but I think copy pasting this is easy enough for now.